.version

```diff
@@ -1 +1 @@
-7.4.0-SNAPSHOT
+7.5.1
```

Comment:
Not sure why `paywalls` doesn't have the right version name here but I will make sure it gets updated when making the release